src/components/PlainText/styles.ts

```diff
@@ -7,7 +7,7 @@ type Styles = {
 
 const styles = StyleSheet.create<Styles>({
   text: {
-    fontFamily: fonts.MainMedium
+    fontFamily: fonts.MainRegular
```

Comment:
we have PlainText for this, dont we?

Reply:
How I noticed, main `fontWeight` of out layout is `MainRegular`.

Reply:
mb we will use it in the future, mb it will be good idea to create type for PlainText